ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -101,6 +101,13 @@ type ipfsStream struct {
 	Protocol string
 }
 
+// ipfsRepoGC contains `ipfs repo gc` api response for a single garbace
+// collected content.
+type ipfsRepoGC struct {
+	Key   cid.Cid `json:"Key"`
```

Comment:
Decapitalise the json struct tags

Reply:
IPFS uses capitalized.

Reply:
Maybe, simply don't put a json tag when it does nothing.